src/routes/OverviewPage/ProductDocumentationCard.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import {
+  Button,
+  ButtonVariant,
+  Card,
+  CardBody,
+  CardFooter,
+  CardHeader,
+  CardTitle,
+  Flex,
+  FlexItem,
+  Icon,
+  Label,
+  Text,
+  TextContent,
+  TextVariants,
+} from '@patternfly/react-core';
+import { ExternalLinkAltIcon } from '@patternfly/react-icons';
+
+function ProductDocumentationCard() {
+  return (
+    <Card className="pf-v5-u-h-100">
+      <CardHeader>
+        <CardTitle>
+          <Flex direction={{ default: 'column' }}>
+            <FlexItem spacer={{ default: 'spacerSm' }}>
+              <TextContent>
+                <Text component={TextVariants.h2}>
+                  ACS product documentation
+                </Text>
+              </TextContent>
+            </FlexItem>
+            <FlexItem>
+              <Label isCompact color="green">
+                Documentation
+              </Label>
+              <Text
+                component={TextVariants.small}
+                className="pf-v5-u-color-200 pf-u-ml-sm"
+              >
+                access.redhat.com
+              </Text>
```

Comment:
Even though it will be redundant with the link below, I think this should be a hyperlink, too.

Reply:
discussed w Doron to remove the link